docs/content/en/integrations/social-authentication.md

```diff
@@ -140,12 +143,16 @@ carries the same attributes and a similar procedure. Follow along below.
     DD_SOCIAL_AUTH_OKTA_OAUTH2_API_URL=(str, 'https://{your-org-url}/oauth2/default'),
     {{< /highlight >}}
 
-If during the login process you get the following error: *The
-'redirect_uri' parameter must be an absolute URI that is whitelisted
-in the client app settings.* and the `redirect_uri` HTTP
-GET parameter starts with `http://` instead of
-`https://` you need to add
-`SOCIAL_AUTH_REDIRECT_IS_HTTPS = True` in the settings.
+If you are using an Okta Workforce (not Developer) tenant then use only "/oauth2" for the API URL,  do not include the "default" 
+because your tenant has only one authorization server.
+
+For production deployments where you are using HTTPS to access DefectDojo, also add/set the following.  Note that this is not a "DD_*" 
+environment variable so it is further down in the the settings file.
+
+    {{< highlight python >}}
+    SOCIAL_AUTH_REDIRECT_IS_HTTPS = True
+    {{< /highlight >}}
+
```

Comment:
I suggest to add this as a `DD_` variable and set it default to `False` as the defautl docker-compose deployment has no https. Is that something you could add to the PR?

Reply:
I'm happy to take a crack at that.  I searched through this repo and I only see `SOCIAL_AUTH_REDIRECT_IS_HTTPS` mentioned in this documentation.   Would the change you propose be to [godojo](https://github.com/DefectDojo/godojo) within env.go ?   I believe it is what generates `settings.dist.py`.

I see where the `DD_SOCIAL_AUTH_REDIRECT_IS_HTTPS`  would be added to the template but haven't yet found the  generation of the remainder of the file.

Reply:
Hi @hkelley 

`SOCIAL_AUTH_REDIRECT_IS_HTTPS` is from the [social-auth]( https://python-social-auth.readthedocs.io/en/latest/configuration/settings.html#processing-redirects-and-urlopen) package we use for SSO. 

The change would need to go into `settings.dist.py`.